include/crocoddyl/core/costs/residual.hxx

```diff
@@ -60,28 +60,28 @@ void CostModelResidualTpl<Scalar>::calcDiff(const boost::shared_ptr<CostDataAbst
   const std::size_t nv = state_->get_nv();
   if (is_ru) {
     data->Lu.noalias() = data->residual->Ru.transpose() * data->activation->Ar;
-    d->Arr_Ru.noalias() = data->activation->Arr * data->residual->Ru;
```

Comment:
Could you provide information about improve performance?
I ask this as the results reported in private where not better (in terms of computation time).

Reply:
Ok, thanks for the info.